### PR TITLE
tests: fix bad email msgs in esearch.mbox

### PR DESCRIPTION
**Pull request by shikharid**
Issue https://github.com/dovecot/imaptest/issues/35

### CODE REVIEW

File: src/tests/esearch.mbox [hunk 1]
```diff
@@ -1,28 +1,42 @@
-From user@domain  Fri Feb 22 17:06:21 2008
+From user@domain  Fri Feb 22 17:06:23 2008
```
Comment (sirainen) [inline src/tests/esearch.mbox:1]: Why did you change the timestamps? I guess it doesn't matter since the tests aren't actually using timestamp searches.

Comment (shikharid) [reply src/tests/esearch.mbox:1]: No reason, just copy/paste. Can revert to older if you want?

Comment (sirainen) [reply src/tests/esearch.mbox:1]: Would be nice to avoid unnecessary changes, causing confusion in history.

Comment (shikharid) [reply src/tests/esearch.mbox]: Done

**Comment by shikharid** [comment]
Off topic, help me out if you can @sirainen :
- I have a job that uses the [static binary here](https://github.com/dovecot/imaptest/releases/tag/latest) to run these imaptests rather than build from scratch
- But I still need to checkout the repo to bring in the tests and provide it as an argument (`./imaptest ... test=tests/`)

Are the tests bundled into the binary? If so, how can I provide them as an arg or lead to binary to use the bundled tests.

**Comment by slusarz** [comment]
> Off topic, help me out if you can @sirainen :
> 
>     * I have a job that uses the [static binary here](https://github.com/dovecot/imaptest/releases/tag/latest) to run these imaptests rather than build from scratch
> 
>     * But I still need to checkout the repo to bring in the tests and provide it as an argument (`./imaptest ... test=tests/`)
> 
> Are the tests bundled into the binary? If so, how can I provide them as an arg or lead to binary to use the bundled tests.

Tests are defined in individual text files.  The binary is just that - a binary that interprets test files.  So they are two distinct items.

I don't see any benefit of creating another package with binary + test files since you can do the same thing with downloading the binary and doing a git checkout of the repository to get the test files.  That is easily scripted.

The intention for imaptest binary is to run on locally defined test files or for doing performance testing.

**Comment by shikharid** [comment]
Can you merge these changes @sirainen ?

**Comment by sirainen** [comment]
Merged. Should sync into github soon.
